hhs_oauth_server/settings/base.py

```diff
@@ -78,6 +80,12 @@
 
 ]
 
+REST_FRAMEWORK = {
+    'DEFAULT_THROTTLE_RATES': {
+        'token': env('TOKEN_THROTTLE_RATE', None),
```

Comment:
This may be a duplicate comment, but can we set the default to something other than None? I'd like to actually return these headers by the end of the sprint.